spi_nand_flash/idf_component.yml

```
@@ -0,0 +1,5 @@
+version: "0.0.3"
+description: Driver for accessing spi nand flash
+url: https://github.com/espressif/idf-extra-components/tree/master/spi_nand_flash
```

```suggestion
url: https://github.com/espressif/idf-extra-components/tree/master/spi_nand_flash
repository: https://github.com/espressif/idf-extra-components.git
issues: https://github.com/espressif/idf-extra-components/issues
```

Please consider adding more metadata fields